CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Default Owners
-* @projectatomic/ai-studio-reviewers
+* @containers/podman-desktop-extension-ai-lab-reviewers
```

Comment:
just a FYI, I fixed the error saying it was invalid as the team was not added to this repository

Reply:
Thank you!!